lib/src/mjpeg.dart

```diff
@@ -39,10 +39,13 @@ class Mjpeg extends HookWidget {
   final bool isLive;
   final Duration timeout;
   final WidgetBuilder? loading;
-  final Widget Function(BuildContext contet, dynamic error, dynamic stack)? error;
+  final Client? httpClient;
+  final Widget Function(BuildContext contet, dynamic error, dynamic stack)?
+      error;
   final Map<String, String> headers;
 
   const Mjpeg({
+    this.httpClient,
```

Comment:
Look, named parameter :)

Reply:
That's the widget, I'm talking about `_StreamManager` in my comment :)